backend-api/package.json

```diff
@@ -8,10 +8,11 @@
     "format:check": "./node_modules/.bin/prettier --check '**/*.ts'",
     "lint": "./node_modules/.bin/eslint ./src/functions",
     "pact": "./node_modules/.bin/jest --testPathPattern=tests/pact",
-    "test": "./node_modules/.bin/jest",
+    "test": "./node_modules/.bin/jest --testPathIgnorePatterns=tests/api",
```

Comment:
I didn't have time to make the API test I wrote to work with the GHA workflow on pull request events so I had to ignore `tests/api` here for now.

Reply:
Failure is due to not having SELF_PUBLIC envVar set. Number of ways we could fix.

Question re this @sandy-may - what tests should run in the pull request gha vs merge to main gha?

Reply:
I'm thinking unit and infra and then the api tests run in the pipeline on a against the deployed stack?